resume_json/resume_cli.py

```diff
@@ -38,6 +40,10 @@ def main():
     args = parsing_arguments()
     file_path = args.dir
     resume_json = ResumeJson()
+    if not os.path.isdir(args.theme_dir):
+        print(f'The theme directory {args.theme_dir} is not a directory...')
+        sys.exit(1)
+
```

Comment:
The None problem was solved throughout 74b116c8a6663bee3bcca29d9ed1b30ce7684512, 954e97dde1dc13cb59c7afd7d30d2a65ce3bb17a and d06580e62ff3735e0163330ab77661735c9528e6. You can't really have anything else than None to define a non-defined theme directory. Any path assignment will trigger checks, even with Pathlib. I do think it's the most pythonic way of managing that variable.

About the theme being served, you are right, but that wasn't the subject of the merge and issue. That may be worth another issue.

If that's okay with you, I do have other ideas to improve your jsonresume/resume-cli python implementation.

Reply:
I agree about using pathlib, I should have used it from the beginning, I think I will do this soon.

About the theme being served, I am not sure what you mean, but right now it seems OK, we serve the even theme if not getting a theme to show, even when getting the theme-dir flag, but we need to check why the warning is not printed (I guess it's because of cherrypy, but didn't really check yet).

About other ideas you have, I would love to hear them and see how we can implement them :)